unittests/test_sanity_functions.py

```diff
@@ -213,247 +214,239 @@ def setUp(self):
                                        'src', 'homer.txt')
 
     def test_assert_true(self):
-        self.assertTrue(sn.assert_true(True))
-        self.assertTrue(sn.assert_true(1))
-        self.assertTrue(sn.assert_true([1]))
-        self.assertTrue(sn.assert_true(range(1)))
-        self.assertRaisesRegex(SanityError, 'False is not True',
-                               sn.evaluate, sn.assert_true(False))
-        self.assertRaisesRegex(SanityError, '0 is not True',
-                               sn.evaluate, sn.assert_true(0))
-        self.assertRaisesRegex(SanityError, r'\[\] is not True',
-                               sn.evaluate, sn.assert_true([]))
-        self.assertRaisesRegex(SanityError, r'range\(.+\) is not True',
-                               sn.evaluate, sn.assert_true(range(0)))
-        self.assertRaisesRegex(SanityError, 'not true',
-                               sn.evaluate, sn.assert_true(0, msg='not true'))
+        assert sn.assert_true(True)
+        assert sn.assert_true(1)
+        assert sn.assert_true([1])
+        assert sn.assert_true(range(1))
+        with pytest.raises(SanityError, match='False is not True'):
+            sn.evaluate(sn.assert_true(False))
+        with pytest.raises(SanityError, match='0 is not True'):
+            sn.evaluate(sn.assert_true(0))
+        with pytest.raises(SanityError, match=r'\[\] is not True'):
+            sn.evaluate(sn.assert_true([]))
+        with pytest.raises(SanityError, match=r'range\(.+\) is not True'):
+            sn.evaluate(sn.assert_true(range(0)))
+        with pytest.raises(SanityError, match='not true'):
+            sn.evaluate(sn.assert_true(0, msg='not true'))
```

Comment:
Since you are leaving a blank line after the with statements maybe you should change this test as well?